src/Mapbender/CoreBundle/Resources/views/Element/legend.html.twig

```diff
@@ -1,4 +1,4 @@
 <div id="{{ id }}" class="mb-element mb-element-legend" title="{{ title |trans }}">
-    <div id="imgtest"></div>
-    <div id="legends"></div>
+  <div id="imgtest"></div>
+  <div id="legends" class="legends"></div>
```

Comment:
Why do we have a class and an id here? I know that the comment above about `imgtest` has nothing to do with this change, it was just a change of the indent, but in this case I think we could really refactor it and remove the DOM id!